pkg/controller/util/utils.go

```diff
@@ -52,3 +52,21 @@ func IsNodeAgentReady(c client.Client) bool {
 	}
 	return conf.Status.NodeAgent.Phase == csiv1.NodeAgentPhaseRunning
 }
+
+func Contains(list []string, s string) bool {
```

Comment:
Is it possible to make these functions generic so if we want to remove an integer from an integers list, we don't need to implement a similar function?

Reply:
not that I know of